<launch>
  <!-- Modifying for willow table height 
    <param name="table_height" value="0.77"/>
-->
  <param name="table_height" value="0.74"/>
    <node pkg="pr2_simple_arm_motions" type="table_frame.py" name="table_frame_watcher" output="screen"/>
</launch>
